docs/README.md

```diff
@@ -160,7 +160,7 @@ You can find previous efs-csi-driver versions' images from [here](https://galler
 
 ### Features
 * Static provisioning - Amazon EFS file system needs to be created manually first, then it could be mounted inside container as a persistent volume (PV) using the driver.
-* Dynamic provisioning - Uses a persistent volume claim (PVC) to dynamically provision a persistent volume (PV). On Creating a PVC, kuberenetes requests Amazon EFS to create an Access Point in a file system which will be used to mount the PV.
+* Dynamic provisioning - Uses a persistent volume claim (PVC) to dynamically provision a persistent volume (PV). On Creating a PVC, kubernetes requests Amazon EFS to create an Access Point in a file system which will be used to mount the PV.
```

Comment:
```suggestion
* Dynamic provisioning - Uses a persistent volume claim (PVC) to dynamically provision a persistent volume (PV). On Creating a PVC, Kubernetes requests Amazon EFS to create an Access Point in a file system which will be used to mount the PV.
```

Reply:
then we should all entries of "kubernetes" change to "Kubernetes" (from Capital)...